app/service/data_svc.py

```diff
@@ -69,6 +69,13 @@ def __init__(self, dao, utility_svc):
             await self.dao.create('core_group_map', dict(group_id=identifier, agent_id=agent[0]['id']))
         return 'Saved %s host group' % name
 
+    async def delete_group(self, group_id):
```

Comment:
can you create a doc string like the """ CREATE """ and """ VIEW """ sections for delete code?